Userland/Utilities/chown.cpp

```diff
@@ -55,7 +54,7 @@ int main(int argc, char** argv)
         if (number.has_value()) {
             new_gid = number.value();
         } else {
-            auto* group = getgrnam(parts[1].characters());
+            auto* group = getgrnam(parts[1].characters_without_null_termination());
```

Comment:
Needs to be null terminated, unless you can implement a `getgrnam` that takes a StringView

Reply:
Updated like this, same for `getpwnam()` above:
```
auto* group = getgrnam(parts[1].to_string().characters());
```

Reply:
Added sycall wrappers for getpwnam and getgrnam.